packages/client-utils/package.json

```diff
@@ -3,15 +3,16 @@
   "version": "0.1.0",
   "description": "Utilities for building Agoric clients",
   "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public"
+  },
   "type": "module",
   "main": "src/main.js",
   "files": [
     "src"
   ],
   "scripts": {
     "build": "exit 0",
-    "prepack": "tsc --build tsconfig.build.json",
-    "postpack": "git clean -f '*.d.ts*' '*.tsbuildinfo'",
```

Comment:
Why not build type def files for this package? Wouldn't NPM consumers have problems with type checking when using it?

Reply:
The package has `.d.ts` which would be overwritten, causing a build error. Motivation is https://github.com/Agoric/agoric-sdk/pull/10422

I've removed it from this PR to move things along